test/stress/n_workers_reputers__stress_test.go

```diff
@@ -438,7 +438,7 @@ func WorkerReputerLoop(
 	workerAddresses := make(map[string]string)
 	reputerAddresses := make(map[string]string)
 
-	approximateBlockTimeSeconds := getApproximateBlockTimeSeconds(m)
+	approximateBlockTime := getApproximateBlockTimeSeconds(m)
```

Comment:
using *Seconds means the time is a Duration. 
Otherwise is a number (ie blocks, or just time without a denom yet, as a number).